src/pheval_exomiser/post_process/post_process.py

```diff
@@ -15,5 +19,6 @@ def post_process_result_format(
         score_name=config.post_process.score_name,
         sort_order=config.post_process.sort_order,
         phenotype_only=phenotype_only,
+        disease_analysis=disease_analysis,
```

Comment:
We should make try to make sure that our interfaces for the runners wont change after that. If we see them changing more often, we may have to move some of this to configuration instead, so that a runner that does not implement the "new" interface can still run with some "default" value (`False`)

Reply:
This will change again as I will be splitting the `phenotype_only` into `gene_analysis` and `variant_analysis` but this shouldn't impact the interfaces too much as I am retrieving these variables from the runner configuration file (`config.yaml`)

Reply:
This seems in any case to be Exomiser specific, right? None of the that affects the pheval codebase.

Reply:
It will change the pheval codebase to some agree but won't alter any processes being made- it's more how it will be utilised by a specific runner

Reply:
I guess you understand my concern here - I am happy for you to go ahead, just be very sensitive to the concept of `backwards compatibility` when extending the CLI in the future! Thats all.